src/components/menu/menu.module.scss

```diff
@@ -0,0 +1,136 @@
+@import "src/styles/utils/variables";
+
+.menu {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+  width: 300px;
+  height: 150px;
+  border: 1px solid transparentize($uc-white, .9);
+  border-bottom: none;
+  border-top-left-radius: 300px;
+  border-top-right-radius: 300px;
+  z-index: 2;
+  overflow: hidden;
+  transform: translateY(0) scale(1);
+  transition: transform .3s ease-in-out;
+
+  &.collapsed {;
```

Comment:
`;` :)

Reply:
this provides extra security. protects from CRSS SCSS Injectors

Reply:
![](https://media1.giphy.com/media/x4bO32XtAQZ9u/giphy.gif?cid=ecf05e477bb23888bd9df396cc49b1577d66dd1224263e08&rid=giphy.gif)

Reply:
Fixed mate bro thanks